bundles/io/pom.xml

```diff
@@ -16,6 +16,7 @@
   <packaging>pom</packaging>
 
   <modules>
+	<module>org.eclipse.smarthome.io.audio</module>
```

Comment:
The whole file should use spaces for indentation. So please keep the current scheme.